### DISCUSSION TITLE
Status / Roadmap of https://github.com/paketo-buildpacks/quarkus

### DISCUSSION THREAD

**Discussion by sebastiangraf** [paketo-buildpacks/feedback#70] [General]
Hi there,
I am looking for a way to build Quarkus-Microservices with the help of Buildpacks. Logically, I stumbled upon https://github.com/paketo-buildpacks/quarkus . However, the status of this buildpack is unclear. It seems to be unreleased and no clear information about the roadmap was found.
Is there any information available when this buildpack will become part of the tiny-builder as denoted in its sourcecode?
Thanks for your support.
Sebastian

**Comment by dmikusa** [comment]
This is a relatively new buildpack, which is why we haven't cut a release yet. I do believe that it works, but I'm not sure off hand the steps to use it. @matejvasek is it OK to cut a 0.1 release of the buildpack? Or were there any other things you wanted to add?

Also, do you recall usage instructions for the buildpack? I'm trying to remember and I think you just need to add the buildpack to the order group, but I can't remember where in the order it needs to reside. Thanks!

**Reply by sebastiangraf** [reply]
From my understanding, this buildpack sets the correct paths for the maven-resulting running jar or executable: nothing which can not be set directly via multiple BP_WHATEVER_ARGs...it would just make building as convenient as with spring boot-modules.

However, I tried to include it together with the tiny-image as checked out git-repository and I must admit that it was not working out of the box. But tbh: I do not know wheather this is an artifact of a new release of quarkus (now resulting in other paths of resulting jars), a missing integration into the java-buildpack or a bug itself...

**Reply by dmikusa** [reply]
That's correct. If you could open an issue under that project, we can go from there and try to track down what's happening. Just add your full build command and the full build output (sanitized if necessary). Thanks!

**Reply by sebastiangraf** [reply]
Done: https://github.com/paketo-buildpacks/quarkus/issues/12
Thanks for looking into it.
